extensions/answer_summarizers/models_test.py

```diff
@@ -33,9 +34,9 @@ def test_requires_override_for_calculation(self):
                 state_answers_dict={})
 
     def test_equality_of_hashable_answers(self):
-        hashable_answer_1 = answer_models._HashableAnswer('answer_1')  # pylint: disable=protected-access
-        hashable_answer_2 = answer_models._HashableAnswer('answer_2')  # pylint: disable=protected-access
-        hashable_answer_3 = answer_models._HashableAnswer('answer_1')  # pylint: disable=protected-access
+        hashable_answer_1 = utils.get_hashable_value('answer_1')
+        hashable_answer_2 = utils.get_hashable_value('answer_2')
+        hashable_answer_3 = utils.get_hashable_value('answer_1')
```

Comment:
Hmm,  this isn't right, this means that you are testing `utils.get_hashable_value` and not `answer_models._HashableAnswer`

Reply:
yeah, the class_hashable answer is used outside in a couple of functions. So should I make it public?

Reply:
sounds good.